### PR TITLE
Update DNAs to use sharded sim2h servers

### PR DESCRIPTION
**Pull request by zo-el**
Updated pointers for:
- happ-store
- holo-hosting-app
- servicelogger 
- holofuel

### Reason for update:
Updated all DNAs to use a default sim2h server for specified DNA.

### CODE REVIEW

File: overlays/holo-nixpkgs/dna-packages/default.nix [hunk 1]
```diff
@@ -6,21 +6,21 @@ let
   happ-store = fetchFromGitHub {
     owner = "holochain";
     repo = "happ-store";
-    rev = "f9c5bb938376780b7e41d3234ff21baa6e04fb59";
-    sha256 = "174nhbbxcajdz8z27fhgs7r1py2ap69i8mkam2bn4pvh4skgabl4";
+    rev = "ca66b4c450c333d3ba7a3590b347754cf2c0c122";
```
Comment (peeech) [inline overlays/holo-nixpkgs/dna-packages/default.nix]: This is a commit on happ-store`s `develop` while all the other commits are from `master`. Shouldn't that be unified as well?

Comment (zo-el) [reply overlays/holo-nixpkgs/dna-packages/default.nix]: That is the default branch.

Comment (zo-el) [reply overlays/holo-nixpkgs/dna-packages/default.nix]: Ok, I just updated the default branch in happ-store to master and updated.

**Comment by zo-el** [comment]
> The default `sim2h_url` for HPOS should be `ws://public.sim2h.net:9000`. Therefore [this line](https://github.com/Holo-Host/holo-nixpkgs/blob/2020-01-31-update-dna-pkgs/profiles/logical/hpos/default.nix#L158) still needs to be updated to the new value.

Done

**Comment by yegortimoshenko** [comment]
Please review rebased history: 510de08e71643b0da00e3a7cf5b358b5dd7a545f...4b58db1d4de8247a0a1dfb2738c6d7a7cd53ed9d vs 952564c74ea0e2b6fa2af3fc5e48ac2943d2a833...1c7cf4d9ba591f431279399484869928589e91c8.

In particular:
- "update" vs "updated"
- for `overlays/`, part before the colon is derivation attribute name, not path, so `dnaPackages.holofuel` not `overlays/dna-packages`
- best to break bumps into separate commits each so that it's easy to track changes and bisect/revert if necessary
- commit messages are only properly rendered in GitHub if they are shorter than ~75 chars
- for long paths, it's best to leave only the most important/best identifying parts, so `profiles/logical/hpos` is usually too long, but `logical/hpos` is much shorter

I've also changed the PR title so it's obvious what the change is for (sim2h sharding).

Check out <https://nixos.org/nixpkgs/manual/#submitting-changes-making-patches>.

File: profiles/logical/hpos/default.nix [hunk 2]
```diff
@@ -155,7 +155,7 @@ in
       instances = map instanceConfig dnas;
       network = {
         type = "sim2h";
-        sim2h_url = "wss://sim2h.holochain.org:9000";
+        sim2h_url = "ws://public.sim2h.net:9000";
```
Comment (zippy) [inline profiles/logical/hpos/default.nix]: Which conductor uses this sim2h server?  I'm assuming the DNA's should specify their sim2h servers in the DNAs with the sim2h_url property, and this just is a backstop?

Comment (yegortimoshenko) [reply profiles/logical/hpos/default.nix]: My understanding is that it's a backstop, and each DNA comes with hardcoded sim2h URL property: https://github.com/holochain/happ-store/commit/74c257730201890fa0f5b503111b47fc34ffdf7a#diff-4b1eb3dc48c4e16d49db5b42298fe654R19

Comment (peeech) [reply profiles/logical/hpos/default.nix]: Yes, each of bumped DNAs have hard-coded sim2h_url in `manifest.json` under `properties.sim2h_url`. The one above is a default `sim2h_url` which is overridden whenever DNA is presenting its own. It produces the following part of conductor-config.toml:
```
[network]   
sim2h_url = "ws://public.sim2h.net:9000"
type = "sim2h"
```

**Pull request merged**